support-frontend/assets/pages/new-contributions-landing/contributionsLanding.scss

```diff
@@ -950,6 +946,13 @@ form {
 
 /* Server side rendering test loading message*/
 
+
+.gu-content--contribution-form--placeholder {
+  .gu-content__blurb--blurb, .gu-content__blurb--header, .countryGroups__label  {
+    opacity: 0.0;
```

Comment:
`visibility: hidden` is an alternative, when you want to hide something but preserve its space in the layout

Reply:
👍